File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        
        int odd_sum = 0;
        int even_sum = 0;
        
        for (int i = 0; i < num_list.length; i++) {
            if (i % 2 == 1) {
                odd_sum += num_list[i];
            } else {
                even_sum += num_list[i];
            }
        }
        
        if (odd_sum >= even_sum) {
            answer = odd_sum;
        } else {
            answer = even_sum;
        }
        
        return answer;
    }
}

File: JS_Algo/프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
    var answer = 0;
    
    let odd = 0
    let even = 0
    
for (let i = 0; i < num_list.length; i++) {
i % 2 != 0 ? odd += num_list[i] : even += num_list[i]
}
    if (odd >= even) {
        answer = odd
    } else if (odd < even) {
        answer = even
    }
    
    return answer;
}